packages/components/src/TokenTutorial/TokenTutorialTextComponents.tsx

```diff
@@ -112,27 +109,13 @@ export const Topic1Tutorial4: React.SFC = props => (
   <>
     <h2>What are Civil tokens (CVL)?</h2>
     <p>
-      Civil tokens (CVL) enable the “community-owned and operated” Civil model to work. A token is a piece of software,
-      it’s not a physical coin. But it does have value and represents a piece of ownership in the network.
-    </p>
-    <p>
-      CVL is a utility token based on the{" "}
-      <a href="https://theethereum.wiki/w/index.php/ERC20_Token_Standard" target="_blank">
-        ERC20 protocol
-      </a>. It’s a value stored in a decentralized database that’s managed by Civil’s smart contracts, which allows the
-      Civil platform to interact with the Civil token. CVL is the software that bridges Civil with the Ethereum
-      blockchain.
+      Civil tokens (CVL) enable the "community-owned and operated" Civil model to work. A token is a piece of software,
+      it's not a physical coin. But it does have value and represents a piece of ownership in the network.
     </p>
     <p>
-      Buying Civil tokens signals that you wish to participate in and contribute to Civil’s community. Civil tokens
-      unlock specific activities on the Civil platform, including sponsoring or launching a newsroom, challenging and
-      voting for/against Newsrooms for ethics violations or appealing the outcome of a community vote to the Civil
-      Council.
-    </p>
-    <p>
-      There will only ever be a fixed supply of Civil tokens created. Owners of Civil tokens will be economically
-      incentivized to ensure existing and future Civil Newsrooms maintain high quality standards as the network, and its
-      reach, grows.
+      For those that want to get more technical, CVL is a utility token based on the ERC20 protocol. It's a value stored
```

Comment:
I assume this copy has all been vetted, but we say here CVL is a utility token and not a consumer token - which I think is a departure from what we normally say

Reply:
I flagged it in the quip doc and will update if needed